src/app/public/styles/themes/modern/_compat/_mixins.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../variables';
+
+@mixin sky-theme-modern-btn-tab {
+  font-weight: $sky-theme-modern-font-body-default-weight;
+}
+
+@mixin sky-theme-modern-btn-tab-selected {
+  background-color: transparent;
```

Comment:
I might be missing something simple, but shouldn't the selected `background-color` be a blue?

Reply:
Not in the new theme. A selected tab is decorated with a blue bottom border instead.

Reply:
Got it.